libs/sdk-ui-dashboard/src/model/commandHandlers/dashboard/tests/saveAsDashboardHandler.test.ts

```diff
@@ -130,23 +124,18 @@ describe("save as dashboard handler", () => {
         beforeEach(async () => {
             await preloadedTesterFactory((tester) => {
                 Tester = tester;
-            }, LockedDashboardIdentifier);
+            }, SimpleDashboardIdentifier);
         });
 
         const TestDashboardTitle = "My Dashboard Copy";
 
-        it("should save an existing locked dashboard as a copy with isLocked flag cleaned", async () => {
```

Comment:
Should we skip and migrate later?

Reply:
Locked is not supported in this way ... on tiger is done via parent workspace thats why deleted